### PR TITLE
Leaf/1.6 tags bridge tables

### PR DESCRIPTION
**Pull request by sosropartono**
Created SQL queries for tags and tags bridge tables

#125

### CODE REVIEW

File: backend/settings.py [hunk 1]
```diff
@@ -85,8 +85,17 @@
...
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'user-data',
+        'HOST': '127.0.0.1',
+        'PORT': 3306,
+        'USER': 'root',
+        'PASSWORD': 'lmaozedong',
```
Comment (PattyCakesxD) [inline backend/settings.py:93]: Is this password meant to be here? (similar to the other table pr)

**Review by PattyCakesxD — APPROVED** [review]
Just 1 concern, lgtm otherwise

File: mesh/profiles/urls.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from . import views as profile_views
+from django.http import HttpRequest
+
+urlpatterns = [
+    
+    # Home Path for Example API 'exampleapi'
```
Comment (misslame) [inline mesh/profiles/urls.py:7]: This is not the example api urls

File: mesh/profiles/urls.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    path('biography/', profile_views.biography, name="biography"),
```
Comment (misslame) [inline mesh/profiles/urls.py:8]: What is this biography stuff? How is it related to tags table?

**Review by misslame — CHANGES_REQUESTED** [review]

File: mesh/profiles/urls.py (hunk 3 above)
Comment (sosropartono) [reply mesh/profiles/urls.py:8]: Sorry about that, I do not remember replacing it with that... will fix

**Pull request closed without merging**